one-ui/src/test/java/com/marklogic/hub/oneui/managers/SavedQueriesManagerTest.java

```diff
@@ -0,0 +1,136 @@
+package com.marklogic.hub.oneui.managers;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.marklogic.hub.oneui.Application;
+import com.marklogic.hub.oneui.TestHelper;
+import com.marklogic.hub.oneui.exceptions.BadRequestException;
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+@ExtendWith(SpringExtension.class)
+@SpringBootTest(classes = {Application.class})
+public class SavedQueriesManagerTest extends TestHelper {
+
+    @Autowired
+    private SavedQueriesManager savedQueriesManager;
```

Comment:
We don't want to autowire in Manager classes - just create one with the HubConfig in the test framework.